projects/demo/src/modules/components/stepper/stepper.component.ts

```diff
@@ -40,7 +41,7 @@ export class ExampleTuiStepperComponent {
 
     icon = this.iconVariants[0];
 
-    readonly stateVariants = ['normal', 'pass', 'error'] as const;
+    readonly stateVariants = [TuiStepState.Normal, TuiStepState.Pass, TuiStepState.Error];
```

Comment:
I believe we wanted to step away from enums here.

Reply:
fixed